app/pods/contests/college/all/archived/controller.js

```diff
@@ -0,0 +1,52 @@
+import Controller from '@ember/controller';
+import { action, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { restartableTask } from 'ember-concurrency-decorators';
+
+export default class ArchivedController extends Controller {
+  @service store
+
+  queryParams = ['offset', 'limit']
+  offset = 0
+  limit = 6
+  q = ''
+
+  @computed('offset', 'limit')
+  get page() {
+    return {
+      offset: this.offset,
+      limit: this.limit
+    }
+  }
+
+  @computed('q')
+  get filter() {
```

Comment:
return filter only if q !== '' otherwise it will put a filter object in params with value: $ILike: '%%'
which will bring the correct results but psql query will take more time.